### ISSUE TITLE
Use space case for filter options dropdowns

### ISSUE THREAD

**Issue by keyserj** [amelioro/ameliorate#356]
**Is your feature request related to a problem? Please describe.**
In the filter options, the dropdowns with hardcoded values all use camelCase for their options, which is odd for non-developers to read.

These are the offending dropdowns (to see the Detail dropdown, "tradeoffs" must be selected in the Standard Filter):
![image](https://github.com/user-attachments/assets/64a49eae-2935-40f0-86eb-6d3e34bf339f)

- e.g. Standard Filter dropdown has "highLevel"
- e.g. Solution Detail dropdown has "connectedToCriteria"
- e.g. Node Types dropdown has "solutionComponent"

**Describe the solution you'd like**
We should use Space Case instead.

**Describe alternatives you've considered**

**Additional context**
- the Breakdown section lives [here](https://github.com/amelioro/ameliorate/blob/b21fbdc7030c5d5a6afa51dc87dc26333bacdaf0/src/web/view/components/Filter/StandardFilter.tsx)
- the General Filters section lives [here](https://github.com/amelioro/ameliorate/blob/b21fbdc7030c5d5a6afa51dc87dc26333bacdaf0/src/web/view/components/Filter/GeneralFilters.tsx)
- we use Material UI's Autocomplete component for our dropdowns, so [that component's documentation](https://mui.com/material-ui/react-autocomplete/) might be useful

**Technical ideas**
- we could probably just make the default to be Space Case for all dropdowns that don't specify a label for their options

**Comment by diya-kishore** [comment]
Can i work on this if it is still available?

**Comment by keyserj** [comment]
@diya-kishore yep! Let me know if you have any questions. Also I see that I didn't put this in the technical details, but we use Material UI's Autocomplete component for our dropdowns, so [that component's documentation](https://mui.com/material-ui/react-autocomplete/) might be useful for this.

**Comment by diya-kishore** [comment]
Sure. Please add the Hacktoberfest accepted label if possible.I would
appreciate it very much. Thankyou

**Comment by keyserj** [comment]
Hey @diya-kishore thanks for looking at this ticket - I'm going to unassign you to open this ticket for others. Let me know if you're still working on it and I can reassign you.

**Comment by lovelypangotra2003** [comment]
Hey! Could this issue be assigned to me?
Also, have there been any UI/UX changes related to this since this issue was opened?

**Comment by keyserj** [comment]
@lovelypangotra2003 assigned! Surprisingly, no major UX changes related to this since creating the issue. You should be able to find the filters in the Views tab of the side pane (if you're on a > 1500px monitor you need to expand the pane from the upper-right corner). Note that the Solution Detail dropdown only shows conditionally when the "tradeoffs" Standard Filter is selected.

Let me know if you have any questions!

**Comment by keyserj** [comment]
@lovelypangotra2003 that's a good question, thanks for asking. I'd say it depends what you want to put in there. Generally I would try to avoid putting things in `utils` if you already have an idea for how that thing should be categorized. For example, if you're thinking about a string-based util, you could put that right into something like `src/common/string.ts`.

I have noticed that the `src/common` dir is getting a little big, so it might be getting to the point where the files in there should be categorized. Most of those things look like models (corresponding to tables), so ` src/common/models/` dir seems pretty good. You don't have to do that though.
